test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -14218,6 +14218,11 @@ def create_eatcpu_job(self, duration=None, mom=None):
         """
         Create a job that eats cpu indefinitely or for the given
         duration of time
+
+        :param duration: The duration, in seconds, to sleep
+        :type duration: int
+        :param mom: the MoM object on which to execute the job
+        :type mom: str
```

Comment:
My only other comment is that the mom parameter in the rest of this file means a mom object.  Here it is a host name for a mom.  This is existing code, so feel free to do what you want, but I'd either rename this 'host' or allow a mom or str.  Check for the instance of a mom object, and then call mom.shortname() on it.  If you do choose to keep it a string, I wouldn't call this a MoM object.  There is a mom object in this library.

Reply:
I renamed it to hostname as the rest of the functions use hostname.  Just pushed.